front/app/components/admin/GraphCards/PostsByTimeCard/index.tsx

```diff
@@ -6,26 +6,23 @@ import usePostsByTime from './usePostsByTime';
 // components
 import GraphCard from 'components/admin/GraphCard';
 import { Box } from '@citizenlab/cl2-component-library';
+import Title from './Title';
 import Chart from './Chart';
```

Comment:
I really want to get rid of these 'Graph wrappers' as soon as possible, so rewrote them to use our component library components